src/main/java/de/tum/in/www1/artemis/web/rest/TextAssessmentResource.java

```diff
@@ -234,21 +235,21 @@ public TextAssessmentResource(AuthorizationCheckService authCheckService, TextAs
     }
 
     /**
-     * Cancel an assessment of a given submission for the current user, i.e. delete the corresponding result / release the lock. Then the submission is available for assessment
+     * POST exercise/:exerciseId/submission/:submissionId/cancel-assessment : Cancel an assessment of a given submission for the current user, i.e. delete the corresponding result / release the lock. Then the submission is available for assessment
      * again.
      *
      * @param submissionId the id of the submission for which the current assessment should be canceled
      * @param exerciseId the exerciseId of the exercise for which the assessment gets canceled
      * @return 200 Ok response if canceling was successful, 403 Forbidden if current user is not the assessor of the submission
      */
-    @PutMapping("/exercise/{exerciseId}/submission/{submissionId}/cancel-assessment")
+    @PostMapping("/exercise/{exerciseId}/submission/{submissionId}/cancel-assessment")
```

Comment:
remove the "/" in the beginning. Make it consistent with other URLs in this resource